### PR TITLE
Add RUN_NODE_CACHE_PATH and RUN_NODE_ERROR_MSG variables

### PR DESCRIPTION
**Pull request by nmssilva**
- Customizing cache path and error message is now possible. I chose to add environment variables for this solution.
    - I also thought about having a configuration file where these variables would be defined, otherwise defining these variables has to be done by editing ```~.bashrc```  or ```~/.bash_profile```, but it didn't sound a big of a problem to edit this files, and adding another file to the project would just increase complexity in the script, so i kept it simple.

- The readme file has been updated explaining the usage for this. If something isn't explicit let me know.

### CODE REVIEW

File: run-node [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
+fi
 
 get_user_path() {
 	[[ -x "/usr/libexec/path_helper" ]] && eval $(/usr/libexec/path_helper -s)
 	echo "$($SHELL -i -l -c 'echo -e "\n"PATH=\"$PATH:\$PATH\""\n"' 2>/dev/null | grep "^PATH=")" > "$PATH_CACHE"
 }
 
 set_path() {
-	if [[ -f "$PATH_CACHE" ]]; then
+	if [ -f "$PATH_CACHE" ]; then
```
Comment (sindresorhus) [inline run-node]: Don't do unrelated changes

File: run-node [hunk 2]
```diff
@@ -37,5 +41,9 @@ fi
 if has_node; then
 	node "$@"
 else
-	echo "Couldn't find the Node.js binary. Ensure you have Node.js installed. Open an issue on https://github.com/sindresorhus/run-node"
+	if [ -z $RUN_NODE_ERROR_MSG ]; then
```
Comment (sindresorhus) [inline run-node]: Should use double brackets.

File: readme.md [hunk 3]
```diff
@@ -46,6 +46,23 @@ $ curl -sSLO https://github.com/sindresorhus/run-node/raw/master/run-node && chm
 ```
 ./run-node file.js
 ```
+### Add and edit customizable cache path and error message
+
+The cache path and error message are defined by ```RUN_NODE_CACHE_PATH``` and ```RUN_NODE_ERROR_MSG``` respectively. To add and edit this variables add this line to ```~.bashrc``` or ```~/.bash_profile```:
```
Comment (sindresorhus) [inline readme.md]: You can use single backticks instead of 3.

Comment (sindresorhus) [reply readme.md]: And this is not just meant for your bashrc, but also for scripts that want to use set a different cache path or message. This should be mention too.

File: readme.md [hunk 4]
```diff
@@ -46,6 +46,23 @@ $ curl -sSLO https://github.com/sindresorhus/run-node/raw/master/run-node && chm
...
+
+```
+#node-run variables
```
Comment (sindresorhus) [inline readme.md]: This is not necessary.

File: readme.md [hunk 5]
```diff
@@ -46,6 +46,23 @@ $ curl -sSLO https://github.com/sindresorhus/run-node/raw/master/run-node && chm
...
+export RUN_NODE_ERROR_MSG="Custom error message"
+export RUN_NODE_CACHE_PATH="/custom/path/to/cache"
+```
+
+For example:
```
Comment (sindresorhus) [inline readme.md]: Should mention that the below is the default.

**Review by sindresorhus — CHANGES_REQUESTED** [review]

File: readme.md (hunk 3 above)
Comment (nmssilva) [reply readme.md]: Your readme used 3 backticks instead of 1 so I kept like it was...

Comment (nmssilva) [reply readme.md]: I will mention that it is also meant for scripts

Comment (mafredri) [reply readme.md]: You should use single backticks for inline code (e.g. in a sentence) and triple backticks for code blocks, see https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting.

Comment (nmssilva) [reply readme.md]: Ah, I see, sorry I did not know that... I will update this in the next hour.

**Comment by nmssilva** [comment]
I have made the requested changes. Please review and let me know if there is something wrong.

**Comment by nmssilva** [comment]
Commenting just to remember that I've made the changes and I'm waiting for review.

File: readme.md [hunk 6]
```diff
@@ -46,6 +46,17 @@ $ curl -sSLO https://github.com/sindresorhus/run-node/raw/master/run-node && chm
...
 ./run-node file.js
 ```
+### Add and edit customizable cache path and error message
+
+The cache path and error message are defined by `RUN_NODE_CACHE_PATH` and `RUN_NODE_ERROR_MSG` respectively. To add these variables to `~.bashrc` or `~/.bash_profile`. This variables can also be used in scripts that want to use set a different cache path or message.
+
+Default configuration:
+
+```
+#node-run variables
```
Comment (satazor) [inline readme.md]: I think this line is not necessary and @sindresorhus already asked to remove it.

**Review by satazor — CHANGES_REQUESTED** [review]

File: readme.md (hunk 6 above)
Comment (nmssilva) [reply readme.md]: Yes that's true. Sorry, I didn't get it the first time... Now I understood :+1: line removed

**Review by sindresorhus — APPROVED** [review]

**Comment by sindresorhus** [comment]
Thanks for your patience. Looks good with some minor readme tweaks :)
